kartei.cls

```diff
@@ -150,7 +150,7 @@
 \PassOptionsToClass{twoside}{scrartcl}%
 \PassOptionsToClass{cleardoublepage=plain,DIV=calc}{scrartcl}%
 \LoadClass{scrartcl}
-\RequirePackage{scrlayer-scrpage}
```

Comment:
Ich glaube man sollte heute eher `scrlayer-scrpage` verwenden, den Wechsel hatte ich vor einer Weile mal gemacht.

Reply:
Da habe ich eigentlich nie etwas angefasst. Wundert mich, dass da etwas farblich markiert ist. Ich verstehe aber auch gerade nicht was da geändert werden müsste.

Reply:
Dir fehlt glaube ich das neuste Update der Kartei.

Reply:
Ich muss die Tage mal schauen, wie ich so einen Direktvergleich des Codes machen kann. Dann sehe ich ja welche Zeilen eine ggf. gute Ergänzung sind. Ein Problem, welches ich aber vermutlich nicht so einfach lösen kann, ist die Nutzung von Variablen. Da fehlt mir leider das KnowHow für.

Reply:
hm, aber so lange ich nicht sehe, was du nun eigentlich vorhast und geändert hast, kann ich das hier ja gar nicht mit Tipps befeuern (ändern sowieso nicht, da das ja dein Branch ist). So wird das dann leider eher nichts.

Reply:
Jo - da stimme ich dir leider zu. Ich habe damals erstmalig im Internet wohl eine nicht-aktuelle Version gefunden und auf der Basis habe ich eben... rumgefrickelt - das ganze war meine erste Begegnung mit Latex. 

Ok... mein Vorschlag:

- Wir löschen das ganze Zeug hier erstmal, was ich hochgeladen habe.
- Ich schaue zwischen den Jahren, ob ich auf Basis der aktuellen Version hier Änderungen hinbekomme. Falls ja, dann lade ich das hoch. Ich versuche es so gut wie realistisch möglich. Wenn nicht, dann nicht :-) 
- Wenn ich es nicht hinbekomme, melde ich mich nochmal. Dann kannst du sagen, ob du an den Änderungen Interesse hast und ich versuche zumindest so zu beschreiben "was" und "wo vermutlich im Code" eingegriffen werden muss (auch wenn du das vermutlich sowieso fix findest).

Was hältst du davon?

Reply:
Prima, dann schließe doch diesen PR und mach einen neuen auf, wenn du dazu gekommen bist. Vielleicht findest du ja auch die Zeit, dich ein wenig in Variablen reinzudenken, die sind nicht so schwierig aber sehr cool.